File: python/AnggotaDPR.py
class AnggotaDPR:
    __id  = ""
    __name = ""
    __field = ""
    __electoral_district = ""
    __position = ""
    __party = ""



    def __init__(self, id = "", name = "", field = "", electoral_district = "", position = "", party = ""):
        self.__id = id
        self.__name = name
        self.__field = field
        self.__electoral_district = electoral_district
        self.__position = position
        self.__party = party

    def getId(self):
        return self.__id
    
    def getName(self):
        return self.__name
    
    def getField(self):
        return self.__field
    
    def getElectoralDistrict(self):
        return self.__electoral_district
    
    def getPosition(self):
        return self.__position
    
    def getParty(self):
        return self.__party
    
    def setId(self, id):
        self.__id = id

    def setName(self, name):
        self.__name = name

    def setField(self, field):
        self.__field = field

    def setElectoralDistrict(self, electoral_district):
        self.__electoral_district = electoral_district

    def setPosition(self, position):
        self.__position = position

    def setParty(self, party):
        self.__party = party

    

    

    

File: java/AnggotaDPR.java
public class AnggotaDPR {
    private String id;
    private String name;
    private String field;
    private String electoralDistrict;
    private String position;
    private String party;

    public AnggotaDPR(){
        this.id = "";
        this.name = "";
        this.field = "";
        this.electoralDistrict = "";
        this.position = "";
        this.party = "";
    }

    public AnggotaDPR(String id, String name, String field, String electoralDistrict, String position, String party) {
        this.id = id;
        this.name = name;
        this.field = field;
        this.electoralDistrict = electoralDistrict;
        this.position = position;
        this.party = party;

    }

    public String getId() {
        return this.id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return this.name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getField() {
        return this.field;
    }

    public void setField(String field) {
        this.field = field;
    }

    public String getElectoralDistrict() {
        return this.electoralDistrict;
    }

    public void setElectoralDistrict(String electoralDistrict) {
        this.electoralDistrict = electoralDistrict;
    }

    public String getPosition() {
        return this.position;
    }

    public void setPosition(String position) {
        this.position = position;
    }

    public String getParty() {
        return this.party;
    }

    public void setParty(String party) {
        this.party = party;
    }

    
}